crates/orchestrator/src/generators/bootnode_addr.rs

```diff
@@ -0,0 +1,34 @@
+use std::net::IpAddr;
+
+use super::errors::GeneratorError;
+
+pub fn generate(
+    peer_id: &str,
+    ip: &IpAddr,
+    port: u16,
+    args: &[&String],
+    p2p_cert: &Option<String>,
+) -> Result<String, GeneratorError> {
+    let addr = if let Some(index) = args.iter().position(|arg| (*arg).eq("--lister-addr")) {
```

Comment:
Typo ? Could have been catched by tests :stuck_out_tongue_closed_eyes: 

```suggestion
    let addr = if let Some(index) = args.iter().position(|arg| (*arg).eq("--listen-addr")) {
```